src/CMakeLists.txt

```diff
@@ -186,6 +187,11 @@ endif()
 add_subdirectory(autotype)
 add_subdirectory(cli)
 
+add_subdirectory(sshagent)
+if(WITH_XC_SSHAGENT)
+    set(sshagent_LIB sshagent Qt5::Core Qt5::Network)
```

Comment:
I am removing this from the HTTP rewrite because its unnecessary. You just need to use "sshagent" in the target_link_libraries for keepassxc_core below.

Reply:
How do you specify it as an optional requirement if it's not inside a cmake variable?